src/components/cards/remote-cards/RemoteCard.tsx

```diff
@@ -241,6 +240,7 @@ export const RemoteCard: React.FC<RemoteCardProps> = ({ id, gutterSize, carousel
           </Column>
         )}
       </Columns>
+      <Border borderRadius={CARD_BORDER_RADIUS} />
```

Comment:
noticed on Android this was wrapping the remote card column itself and not the whole card, probably was the case on iOS too but much more noticeable on Android